app/controllers/challenges_controller.rb

```diff
@@ -19,6 +20,26 @@ def create
     end
   end
 
+  def for_evaluation
+    @evaluable_challenges = Challenge.evaluable(current_user)
+
+    if @evaluable_challenges.any?
+      render :for_evaluation, status: :ok
+    else
+      render json: { resource: 'challenge', errors: [challenge: 'not found'] }, status: :not_found
```

Comment:
Yeah, since you brought it up, sending this as an error doesn't feel right. Having no results doesn't mean an error occurred. I feel like you can get rid of the check altogether. You can show custom error messages with the api util in the UI, but that would give the impression that the user did something wrong when that's not the case